CONTRIBUTING.md

```diff
@@ -347,3 +347,32 @@ cd {PORTAL_ROOT}/modules
 
 # Lastly, share the release information in our public Slack channel!
 ```
+
+## Testing local changes in a local [liferay-portal](https://github.com/liferay/liferay-portal) environment
+
+When working on an issue, you might want to test your changes before submitting
+a pull request.
+
+To test your changes in a liferay-portal (>= 7.3.x) environment you first need
+to link the desired packages.
+
+```bash
+# Let's assume you have made a change in the clay-button component
+
+# From the root of the clay repository, run
+yarn workspace @clayui/button build
+yarn workspace @clayui/buttom link
+
+# Note that if you have linked this package before, yarn will print a warning
+# but that doesn't mean that this step failed
+
+# Now navigate to the frontend-taglib-clay module
+cd {PORTAL_ROOT}/modules/apps/frontend-taglib/frontend-taglib-clay
+
+# Link the @clayui/button package
+yarn link @clayui/button
+
+# Deploy the module
+gradlew deploy -a
```

Comment:
@julien maybe add the detail that it might fail and have to take down the portal and start it again because of the module.

Reply:
@matuzalemsteles yes good idea. I just want to find a "nice" way of saying it, because this is something that wasn't happening before.